prime-router/src/main/kotlin/fhirengine/utils/HL7Reader.kt

```diff
@@ -293,7 +297,7 @@ class HL7Reader(private val actionLogger: ActionLogger) : Logging {
                     DefaultHapiContext(
                         ParserConfiguration(),
                         ValidationContextFactory.noValidation(),
-                        ReportStreamCanonicalModelClassFactory(ORU_R01::class.java),
+                        ReportStreamCanonicalModelClassFactory(RadxMarsORU_R01::class.java),
```

Comment:
Subbing this out since it's just additive on top of the existin 2.7 and will ultimately become the FhirInventory ORU message